drivers/eeprom/eeprom_st25dv.c

```diff
@@ -0,0 +1,970 @@
+/*
+ * Copyright (c) 2023 Kickmaker
+ * Copyright (c) 2021 Martin Schröder <info@swedishembedded.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT st_st25dv
+
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/eeprom.h>
+#include <zephyr/drivers/eeprom/eeprom_st25dv.h>
+#include <soc.h>
+
+#define LOG_LEVEL CONFIG_EEPROM_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(eeprom_st25dv);
```

Comment:
```suggestion
#include <zephyr/logging/log.h>
LOG_MODULE_REGISTER(eeprom_st25dv, CONFIG_EEPROM_LOG_LEVEL);
```

Reply:
Updated. All other eeprom drivers seem to use the same way as I used before.